habitat_baselines/rl/ddppo/policy/resnet_policy.py

```diff
@@ -367,6 +369,29 @@ def __init__(
 
             rnn_input_size += hidden_size
 
+        if InstanceImageGoalSensor.cls_uuid in observation_space.spaces:
```

Comment:
Let's avoid copy pasta and keep our code [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) and just add loop to line L:351:

```
for uuid in [InstanceImageGoalSensor.cls_uuid, ImageGoalSensor.cls_uuid]:
     if uuid in observations:
      ...